models.py

```diff
@@ -46,14 +49,63 @@ def forward(self, x):
         return out
 
 
-class Simple_regressor(torch.nn.Module):
+class Simple_regressor(nn.Module):
 
     def __init__(self, in_channels, out_channels=1):
 
         super(Simple_regressor, self).__init__()
 
-        self.layer = torch.nn.Linear(in_channels, out_channels)
+        self.layer = nn.Linear(in_channels, out_channels)
 
     def forward(self, x):
 
         return self.layer(x)
+
+class UNet(nn.Module):
+
+    def __init__(self, n_class):
+        super().__init__()
+                
+        self.dconv_down1 = double_conv(3, 64)
```

Comment:
À modifier pour prendre des images médicales en entrée.
```suggestion
        self.dconv_down1 = double_conv(1, 64)
```